src/convert/transformers/decomposedMetadataTransformer.ts

```diff
@@ -227,14 +228,6 @@ const getDefaultOutput = (component: MetadataComponent): SourcePath => {
   return join(baseComponent.getPackageRelativePath(baseName, 'source'), output);
 };
 
-/** handle wide-open reading of values from elements inside any metadata xml file.
- * Return the value of the matching element if supplied, or defaults `fullName` then `name`  */
-const extractUniqueElementValue = (xml: JsonMap, elementName?: string): string | undefined =>
```

Comment:
moved to a utility module so it can be shared between transformers and finalizers